src/specs/concepts/permissions_concepts.cpt

```diff
@@ -1,16 +1,70 @@
-* From <node> deploy "OrgManager" contract passing "upgradable" address, name it "orgmgr"
-* From <node> deploy "RoleManager" contract passing "upgradable" address, name it "rolemgr"
-* From <node> deploy "NodeManager" contract passing "upgradable" address, name it "nodemgr"
-* From <node> deploy "VoterManager" contract passing "upgradable" address, name it "votermgr"
-* From <node> deploy "PermissionsInterface" contract passing "upgradable" address, name it "interface"
-* From <node> deploy implementation contract passing addresses of "upgradable", "orgmgr", "rolemgr", "accountmgr", "votermgr", "nodemgr". Name this as "implementation"
-* From <node> execute permissions init on "upgradable" passing "interface" and "implementation" contract addresses
-
-# Create permissions config file and write the file to <nodes>
+# Start permission network <networkType> <version> <id> <qip714block>
+* Start a <networkType> <version> Quorum Network, named it <id>, consisting of "Node1,Node2,Node3"
+* Deploy permissions model contracts from "Node1" with <version>
+* Create permissions config file with <version> and write the file to "Node1,Node2,Node3"
+* Restart network <id>
+* Deploy "storec" smart contract with initial value "5" from a default account in "Node1" until block height reaches <qip714block>
+
+# Validate network admin role
+* Validate that org "NWADMIN" is approved, has "Node1" linked and has role "NWADMIN"
+* Validate that org "NWADMIN" is approved, has "Node2" linked and has role "NWADMIN"
+* Validate that org "NWADMIN" is approved, has "Node3" linked and has role "NWADMIN"
+* Check "Node1"'s default account is from org "NWADMIN" and has role "NWADMIN" and is org admin and is active
+
+# Propose new organization to the network <version> <id>
+* Validate that org "NWADMIN" is approved, has "Node1" linked and has role "NWADMIN"
+* Check "Node1"'s default account is from org "NWADMIN" and has role "NWADMIN" and is org admin and is active
+* From "Node1" propose and approve new org "ORG1" with "Node4"'s enode id and "Default" account
+* Start stand alone "Node4"
+* Write "permissionsConfig" to the data directory of "Node4"
+* Restart network <id>
+* Validate that org "ORG1" is approved, has "Node4" linked and has role "ORGADMIN"
+
+# Suspend an org <version> <id>
+* Validate that org "NWADMIN" is approved, has "Node1" linked and has role "NWADMIN"
+* Check "Node1"'s default account is from org "NWADMIN" and has role "NWADMIN" and is org admin and is active
+* From "Node1" propose and approve new org "ORG1" with "Node4"'s enode id and "Default" account
+* Start stand alone "Node4"
+* Write "permissionsConfig" to the data directory of "Node4"
+* Restart network <id>
+* Validate that org "ORG1" is approved, has "Node4" linked and has role "ORGADMIN"
+* From "Node1" suspend org "ORG1", confirm that org status is "PendingSuspension"
+* From "Node1" approve org "ORG1"'s suspension, confirm that org status is "Suspended"
```

Comment:
Minor: should you validate that org status has been updated on every node?

Reply:
Not required as the smart contract calls get executed on all nodes